packages/gatsby-source-filesystem/src/create-remote-file-node.js

```diff
@@ -155,25 +119,6 @@ async function fetchPlaceholder({ fromPath, url, cache, ext, name }) {
  * Index of promises resolving to File node from remote url
  */
 const processingCache = {}
-/**
- * pushTask
- * --
- * pushes a task in to the Queue and the processing cache
- *
- * Promisfy a task in queue
- * @param {CreateRemoteFileNodePayload} task
- * @return {Promise<Object>}
- */
-const pushTask = task =>
-  new Promise((resolve, reject) => {
-    queue.push(task, (err, node) => {
-      if (!err) {
-        resolve(node)
-      } else {
-        reject(`failed to process ${task.url}\n${err}`)
```

Comment:
This change actually moves this error guard from https://github.com/gatsbyjs/gatsby/blob/7ab2f4b422b9b807958df2473a68aeb5946037ec/packages/gatsby-source-filesystem/src/create-remote-file-node.js#L52-L103 into the actual fetching.

I think this is fine for most of the code and wonder if `GATSBY_EXPERIMENTAL_REMOTE_FILE_PLACEHOLDER ` is actually used? (Can't find a reference to in either in Google nor in the code base)
 
I think we should guard this again or remove the flag.

Reply:
we can remove it, we added it to do some performing testing but we should remove it.

Reply:
Removed!